server/controllers/getSummaryStats_controller.js

```diff
@@ -61,23 +57,164 @@ exports.allAppSummaries = function(req, res) {
   });
 };
 
+//TODO: MyServerSummary, basically the inverse of myAppSummary
 exports.myServerSummary = function(req, res) {
 //WILL BE FOR INDIVIDUAL SERVERS ON MY SERVER PAGE
-  myServerSums.model.fetchAll()
+//Filter By User ID and SERVER ID and have req.body for time interval (1-30 days)
+  ServerSums.model.fetchAll()
   .then(function(serverSummary) {
-    console.log(serverSummary);
+    // console.log(serverSummary);
     res.send(serverSummary.models);
   })
   .catch(function(error) {
-    console.log('Error getting all Server Summaries', error);
+    console.log('Error getting server summary', error);
     res.send(500);
   });
 };
 
+exports.myAppSummary = (req, res) => {
+//update to req.body variables below
+  var userId = 1;
+  var appId = 1;
+  var days = 2;
+  //appRoutes defined in this scope so servers can cross reference and total up just the hits for this app.
+  //Assumes apps do not share route names, only way to pull out app hits for each server
+  //Server summaries is totaled by route not by app (so would need to let customers know routes must be unique across apps)
+  var appRoutes = {};
+ AppSums.model.where({'users_id': userId, 'appid': appId })
+ .where(knex.raw("created_at > (NOW() - INTERVAL '" + (days * 24) + " hour'" + ")"))
+  .fetchAll()
+  .then(function(appSummary) {
+    appModels = appSummary.models;
+    //Object to keep track of total hits for each day
+    var appTotalHits = {};
```

Comment:
could you make this an array objects with the format [ {date: 20160508, value: 5739}, {date: 20160509, value: 8572}]
this way, i sort by the date property easily.